mySimpleComputer/sc_memoryGet.c

```diff
@@ -0,0 +1,16 @@
+#include "../include/mySimpleComputer.h"
+#include "./sc_constants.h"
+#include "sc_variables.h"
+
+int sc_memoryGet(int address, int* value)
+{
+    if (address < RAM_MIN_ADDRESS || address > RAM_MAX_ADDRESS)
+    {
+        return ERROR;
+    }
+    else
```

Comment:
Предлагаю убрать else, чтобы не было лишней вложенности:
```suggestion
     if (address < RAM_MIN_ADDRESS || address > RAM_MAX_ADDRESS)
    {
        return ERROR;
    }
    *value = RAM[address];
    return OK;
```

Reply:
Пример коммента